pyvista/core/objects.py

```diff
@@ -378,6 +378,29 @@ def plot(self, *args, **kwargs):
         """Plot the texture as image data by itself."""
         return self.to_image().plot(*args, **kwargs)
 
+    @property
+    def cube_map(self):
+        """Return ``True`` if cube mapping is enabled and ``False`` otherwise.
+
+        Is this texture a cube map, if so it needs 6 inputs, one for
+        each side of the cube. You must set this before connecting the
+        inputs.  The inputs must all have the same size, data type,
+        and depth.
+        """
+        return bool(self.GetCubeMap())
```

Comment:
I already get a bool on vtk 9.0.1. Is this perhaps version-dependent?
```suggestion
        return self.GetCubeMap()
```